lib/files.js

```diff
@@ -275,6 +275,23 @@ function getComponentPackage(name) {
   return componentPath && tryRequireEach([componentPath + '/package.json']);
 }
 
+/**
+ * Returns a promise representing the retrieval of content from a file
```

Comment:
Good description, but just needs to follow JSDoc formatting:
http://usejsdoc.org/